src/components/capacity-breakdown/capacity-breakdown-card.tsx

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Select, SelectProps } from '@patternfly/react-core';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  usePrometheusPoll,
+} from '@console/dynamic-plugin-sdk/internalAPI';
+import { BreakdownCardBody } from '../breakdown-card/breakdown-body';
+import { getStackChartStats, sortInstantVectorStats } from '../breakdown-card/utils';
+import { getSelectOptions } from '../breakdown-card/breakdown-dropdown';
+import './capacity-breakdown-card.scss';
+import { humanizeBinaryBytes } from '../../humanize';
+import { BreakdownQueryMapODF } from '../../constants/queries';
+import { PROJECTS, STORAGE_CLASSES, PODS } from '../../constants/index';
+import { 
+  getInstantVectorStats,
+} from '../../selectors/promethues-utils';
+import { parseProps } from '../../selectors/index';
+
+const keys = [PROJECTS, STORAGE_CLASSES, PODS];
+const breakdownSelectItems = getSelectOptions(keys);
+
+const BreakdownCard: React.FC<any> = (props) => {
+  const {name} = parseProps(props);
+  
+  const [metricType, setMetricType] = React.useState(PROJECTS);
+  const [isOpenBreakdownSelect, setBreakdownSelect] = React.useState(false);
+  const { model, metric, queries } = BreakdownQueryMapODF(name, metricType);
+  const queryKeys = Object.keys(queries);
+
+  const [byUsedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[0]],
+    endpoint: "api/v1/query" as any,
+  });
+  
+  const [totalUsedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[1]],
+    endpoint: "api/v1/query" as any,
+  });
+  const metricTotal = _.get(totalUsedmetric, 'data.result[0].value[1]');
+
+  const [usedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[2]],
+    endpoint: "api/v1/query" as any,
+  });
+  const flashsystemUsed =  _.get(usedmetric, 'data.result[0].value[1]');
```

Comment:
just a thought, can we use something similar to this, will this work ??
```
queryKeys.forEach((q) => usePrometheusPoll({
    query: queries[q],
    endpoint: "api/v1/query" as any,
  });

const results = queryKeys.map((q) => {
     const value = _.get(q, 'data.result[0].value[1]');
     return value ? value : q;
});
```

Reply:
Tried, the metric result is undefined.

Reply:
ack !! I am not sure why it is undefined, we poll similarly in our console code too.
Okay, these things can be refactored later also I guess, if need arises.